spec/frontend_generators/assets_copier_spec.rb

```diff
@@ -1,12 +1,60 @@
 require 'spec_helper'
+require 'fileutils'
 
-module FrontendGenerators; describe AssetsCopier do
+module FrontendGenerators
+  describe FrontendGenerators::AssetsCopier do
 
-  context ".copy_assets" do
-    it "copies the assets in a new directory" do
-      copier = AssetsCopier.new(:bootstrap)
-      copier.copy_assets
+    describe '#bootstrap' do
+      before do
+        FileUtils.rm_rf("#{Turf.find(:root)}/tmp/public")
+        FileUtils.rm_rf("#{Turf.find(:root)}/tmp/vendor")
+        copier = AssetsCopier.new(:bootstrap)
+        copier.copy_assets
+      end
+
+      context ".copy_assets" do
+        it "creates the bootstrap files" do
+          expect(File.file?("#{Turf.find(:root)}/tmp/vendor/assets/stylesheets/bootstrap.css")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/vendor/assets/stylesheets/bootstrap.css")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/public/fonts/glyphicons-halflings-regular.eot")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/public/fonts/glyphicons-halflings-regular.svg")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/public/fonts/glyphicons-halflings-regular.ttf")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/public/fonts/glyphicons-halflings-regular.woff")).to eq(true)
+          expect(File.file?("#{Turf.find(:root)}/tmp/public/fonts/glyphicons-halflings-regular.woff2")).to eq(true)
+        end
+      end
     end
-  end
 
-end; end
+    describe '#clean_blog' do
+      before do
+        FileUtils.rm_rf("#{Turf.find(:root)}/tmp/public")
```

Comment:
I think I added some magic here so you can use `Turf.root` instead of `Turf.find(:root)`